pkg/exchange/bybit/exchange.go

```diff
@@ -389,6 +393,67 @@ func (e *Exchange) QueryTrades(ctx context.Context, symbol string, options *type
 	return trades, nil
 }
 
+/*
+QueryKLines queries for historical klines (also known as candles/candlesticks). Charts are returned in groups based
+on the requested interval.
+
+A k-line's start time is inclusive, but end time is not(startTime + interval - 1 millisecond).
+e.q. 15m interval k line can be represented as 00:00:00.000 ~ 00:14:59.999
+*/
+func (e *Exchange) QueryKLines(ctx context.Context, symbol string, interval types.Interval, options types.KLineQueryOptions) ([]types.KLine, error) {
+	req := e.client.NewGetKLinesRequest().Symbol(symbol)
+	intervalStr, err := toLocalInterval(interval)
+	if err != nil {
+		return nil, err
+	}
+	req.Interval(intervalStr)
+
+	limit := uint64(options.Limit)
+	if limit > defaultKLineLimit || limit <= 0 {
+		log.Debugf("limtit is exceeded or zero, update to %d, got: %d", defaultKLineLimit, options.Limit)
+		limit = defaultKLineLimit
+	}
+	req.Limit(uint64(options.Limit))
```

Comment:
We will assign `defaultKLineLimit` when the `options.Limit` is invalid, so we have to set the limit by `req.Limit(limit)`.